packages/gatsby/src/utils/dev-ssr/develop-html-route.ts

```diff
@@ -52,16 +52,112 @@ export const route = ({ app, program, store }): any =>
           column: error.row,
         },
       })
-      let errorHtml = `<title>Develop SSR Error</title><h1>Error<h1>
-        <h2>The page didn't SSR correctly</h2>
+      let errorHtml = `<head>
```

Comment:
I didn't notice this before (only when I saw <head> being added here I looked more), but shouldn't we wrap everything in `<html>` tag? It apparently works as-is so not blocking just 🤔  moment

Reply:
heh — the html spec is very forgiving & I'm a minimalist 😇